src/saving/serializers/ThriveTypeConverter.cs

```diff
@@ -32,10 +32,8 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
         {
             return ThriveJsonConverter.Instance.SerializeObject(value);
         }
-        else
-        {
-            throw new NotSupportedException();
-        }
+
+        throw new NotSupportedException();
```

Comment:
This might be me not understanding how this part of the code works, but wouldn't an exception be called every time this method is used without the else statement?

Reply:
There's a return inside the if. It seems that the code cleanup tool really prefers early exits instead of long if else chains, which I don't mind so I accepted those changes and didn't try to tweak the tool to leave those alone.